apps/service_profile/lib/profile/reducer/bounding_box.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Profile.Reducer.BoundingBox do
+  defstruct [:latitude_path, :longitude_path, :xmin, :ymin, :xmax, :ymax]
```

Comment:
are we skipping `:zmin` and `:zmax` for now?

Reply:
according to johnson, yes